tools/cldr-apps/src/main/webapp/js/esm/cldrGenericVue.js

```diff
@@ -38,7 +49,7 @@ function errorHandler(err) {
 }
 
 function getHtml(json) {
-  let html = cldrStatus.logoIcon();
+  let html = "";
   return html;
```

Comment:
delete getHtml if is serves no purpose?

Reply:
@btangmu I was trying to keep the churn as small as possible on the cldr*.js files. Removing the 'loader' caused a startup failure.  but, I'll keep cutting away at it.

Also, `cldrStatus.logoIcon()` could actually be accessed via the exports now. I'll try that.